Formula/xtrans.rb

```diff
@@ -38,6 +39,7 @@ def install
       --enable-docs=#{build.with?("docs") ? "yes" : "no"}
     ]
 
+    inreplace "Xtranslcl.c", "# include <sys/stropts.h>", "# include <sys/ioctl.h>"
```

Comment:
Please add a comment above this line, such as `# Fedora systems do not provide sys/stropts.h`.
```suggestion
    # Fedora systems do not provide sys/stropts.h
    inreplace "Xtranslcl.c", "# include <sys/stropts.h>", "# include <sys/ioctl.h>"
```